src/main/kotlin/com/openlattice/chronicle/storage/PostgresColumns.kt

```diff
@@ -19,5 +19,8 @@ class PostgresColumns {
         val FIRST_NAME = PostgresColumnDefinition("first_name", PostgresDatatype.TEXT)
         val LAST_NAME = PostgresColumnDefinition("last_name", PostgresDatatype.TEXT)
         val DATE_OF_BIRTH = PostgresColumnDefinition( "dob", PostgresDatatype.DATE )
+        val TUD_ID = PostgresColumnDefinition("tud_id", PostgresDatatype.UUID).notNull()
+        val SUBMISSION_DATE = PostgresColumnDefinition("submission_date", PostgresDatatype.TIMESTAMPTZ)
```

Comment:
notNull with default of now()? Should save a parameter in your SQL query as well. If capturing timezone of submission is important that should be a separate field.

Reply:
I took your suggestion to remove timezone from this column, but did not add it as a separate field. It turns out that `submission_date` values will always be in PST regardless of which timezone it was submitted in since `submission_date` is populated using `now()` on the server clock. 

However timezone does matter when GETting data. If a user tries to get submissions within a datetime range but forgets to account for a timezone difference, they may be confused when unexpected submissions are returned. To make dealing with that easier I added an optional `zoneOffset` field on the `getSubmissionsByDate` endpoint (in commit 099b32d). Though because that endpoint is mainly used to populate the frontend, it's rare that zoneOffset will be needed.

Reply:
Also just noticed this-- submission_date will always be UTC on our servers. I think PT locally. Good thinking on adding the zone offset field. An easier way to handle this is to just accept an OffsetDateTime in the request pojo (will include time zone) and let the frontend handle passing in a full qualified date time. When the query is submitted to postgres it will be transparently converted to UTC time as long the fields you are doing a comparison are TIMESTAMP_TZ

Reply:
Good point regarding how postgres handles timezone conversions (and that the server is on UTC as opposed to PT). Knowing that, it makes sense to modify the endpoint to take OffsetDateTime instead of using a zone offset.